samples/docker/online-default/rke2-sample.yaml

```diff
@@ -43,15 +43,24 @@ metadata:
   namespace: ${CABPR_NAMESPACE}
 spec: 
   replicas: ${CABPR_CP_REPLICAS}
+  version: ${KUBERNETES_VERSION}+rke2r1
+  registrationMethod: control-plane-endpoint
+  rolloutStrategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxSurge: 1
   agentConfig:
-    version: ${KUBERNETES_VERSION}+rke2r1
+    nodeAnnotations:
+      test: "true"
   serverConfig:
-    cni: calico
+    disableComponents:
+      kubernetesComponents:
+      - cloudController
   infrastructureRef:
     apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
     kind: DockerMachineTemplate
     name: controlplane
-  nodeDrainTimeout: 2m
+  nodeDrainTimeout: 30s
```

Comment:
```suggestion
```
It is best to allow draining to take longer, and prevent force removal of etcd members until the draining is completed. Or give it a longer timeout to show the option is there.